test/Google.Bigquery.V2.IntegrationTests/personsData.json

```diff
@@ -0,0 +1,3 @@
+{ "fullName": "John Doe", "age": 22, "gender": "Male", "phoneNumber": { "areaCode": "206", "number": "1234567"}, "children": [{ "name": "Jane", "gender": "Female", "age": "6"}, {"name": "John", "gender": "Male", "age": "15"}], "citiesLived": [{ "place": "Seattle", "yearsLived": ["1995"]}, {"place": "Stockholm", "yearsLived": ["2005"]}]}
```

Comment:
I think it's okay where it is - while we *could* create a resources directory, I don't think I've seen other projects doing that... typically the resources are alongside source code.